src/app.js

```diff
@@ -0,0 +1,64 @@
+import "regenerator-runtime/runtime";
+import axios from "axios";
+
+const BASE = "https://api.github.com/orgs/stackbuilders/repos";
+
+const getAllItems = async () => {
+  try {
+    const response = await axios.get(BASE);
+
+    const items = someItems(response.data);
+
+    console.log(moreThanFive(items));
+    console.log(lastFiveUp(items));
+    console.log(sumStars(items));
+  } catch (errors) {
+    console.error(errors);
+  }
+};
+getAllItems();
+// const repos = getAllItems()
+//   .then((result) => console.log(result))
+//   .catch((err) => {
+//     console.error(err);
+//   });
+const someItems = (items) => {
+  const container = [];
+
+  items.map((item) =>
+    container.push({
+      name: item.name,
+      stars: item.stargazers_count,
+      date: item.updated_at,
+    })
+  );
+  return container;
+};
```

Comment:
I think you don't actually need the `container` array here. You can use the map directly as the return of this function, like:
`
const someItems = (items) => items.map((item) => ({name: item.name, ....etc}))
`